backend/src/psihofon/models/organization.py

```diff
@@ -4,15 +4,15 @@
 
 class Organization(models.Model):
     class Meta:
-        verbose_name = _('Organization')
-        verbose_name_plural = _('Organizations')
+        verbose_name = _('Organizacija')
+        verbose_name_plural = _('Organizacije')
 
     name = models.CharField(
-        verbose_name=_('name'),
+        verbose_name=_('naziv'),
         max_length=255,
     )
     city = models.CharField(
-        verbose_name=_('city'),
+        verbose_name=_('grad'),
         max_length=255,
     )
     website_url = models.URLField(
```

Comment:
Can we add translated verbose name for this as well?

Reply:
done